settings.gradle

```diff
@@ -16,6 +16,16 @@
  * limitations under the License.
  */
 
+plugins {
+  // Enable publishing build scans
+  id "com.gradle.enterprise" version "3.4.1" apply false
```

Comment:
Why does this move to settings.gradle?

Reply:
This is how the plugin is applied in Gradle 6.x. See the manual here: https://docs.gradle.com/enterprise/gradle-plugin/#applying_the_plugin